docs/semgrep-ci/configuration-reference.md

```diff
@@ -22,124 +22,243 @@ Object.entries(frontMatter).filter(
 
 # Continuous Integration (CI) configuration reference
 
-Configure Semgrep CI by passing various environment variables in your Continuous Integration (CI) jobs.
+Use this reference to configure Semgrep's behavior in CI environments. This reference includes both command-line options and environment variables. You can also define these variables within a CI configuration file. Refer to your CI provider's documentation for the correct syntax. Examples are written in a Bash environment except when otherwise stated.
 
-:::info
-While environment variables are the preferred way to configure Semgrep CI, pass any of these options as command-line options. Refer to the output of `semgrep ci --help` to find the corresponding flags.
+:::tip
+Unless otherwise stated, you can also set many of these environment variables within your local environment for testing or experimental purposes. Run `semgrep ci` in your command line after setting environment variables to test their effect.
 :::
 
-## Selecting rules to scan with (`SEMGREP_RULES`)
+## Environment variables for configuring scan behavior
 
-```sh
-SEMGREP_RULES="p/security-audit p/secrets"
+These environment variables configure various aspects of your CI job, such as a job's timeout or source of rules.
+
+### `SEMGREP_APP_TOKEN`
+
+Set `SEMGREP_APP_TOKEN` to send findings to Semgrep App and use rules from the Rule Board. You must have a Semgrep App account to use this environment variable. To generate a token, see [Creating a `SEMGREP_APP_TOKEN`](/docs/semgrep-ci/running-semgrep-ci-with-semgrep-app/#creating-a-semgrep_app_token). `SEMGREP_APP_TOKEN` is incompatible with `SEMGREP_RULES`. Do not set `SEMGREP_RULES` environment variable within the same CI job as `SEMGREP_APP_TOKEN`.
```

Comment:
Optional, not blocking: The - `You must (...)` part and `Do not set (...)` maybe we can move them into a prerequisite admonition and caution admonition?

Reply:
Good idea. Done